Helloworld.ts

```diff
@@ -0,0 +1,2 @@
+let message:string="Hello World!";
```

Comment:
You can skip `:string` here, as the type is inferred by the assignment.

Reply:
Yes..I have removed type declaration for message variable